### PR TITLE
updated for #840.

### PR DESCRIPTION
**Pull request by yoshimis**

### CODE REVIEW

File: jp/components/excel_library_using_cells.md [hunk 1]
```diff
@@ -1,38 +1,333 @@
 ---
-title: Excel Library Component - Native Angular | Ignite UI for Angular
-_description: The Ignite UI for Excel Library component ... TODO.
-_keywords: Ignite UI for Angular, Angular, Native Angular Components Suite, Native Angular Controls, Native Angular Components, Native Angular Components Library, Angular Excel Library, Angular Excel Library Example, Angular Excel Library Component, Angular Excel Engine
+Excel ライブラリ コンポーネント - ネイティブ Angular|Ignite UI for Angular
```
Comment (tiliev) [inline jp/components/excel_library_using_cells.md]: Please add `title: ` in front of the title.

Comment (yoshimis) [reply jp/components/excel_library_using_cells.md]: @tiliev title is back! :)

**Comment by zdrawku** [comment]
@yoshimis it looks like the [PR](https://github.com/IgniteUI/igniteui-docfx/pull/874) that I've just tested and merged caused merge conflicts to this one.

**Comment by yoshimis** [comment]
@zdrawku conflicts are resolved.

**Pull request merged**